### PR TITLE
18Carolinas: re-checkin

### PR DESCRIPTION
**Pull request by roseundy**
First checkin for 18Carolinas (OK, technically the 2nd).

Common file changes:
- UI::Hex
- Engine::Tile, Engine::Part::Frame
Add ability to show a 2nd color on a frame.

### CODE REVIEW

**Comment by roseundy** [comment]
updated_map
updated_tiles

File: assets/app/view/game/hex.rb [hunk 1]
```diff
@@ -62,6 +62,19 @@ def render
             points: FRAME_COLOR_POINTS,
           }
           children << h(:polygon, attrs: attrs)
+
+          if (color2 = @tile&.frame&.color2)
+            attrs = {
+              stroke: color2,
+              'stroke-width': FRAME_COLOR_STROKE_WIDTH,
```
Comment (tobymao) [inline assets/app/view/game/hex.rb:69]: we do style strkeWidth: now

Comment (roseundy) [reply assets/app/view/game/hex.rb:69]: OK. Was just cut and pasting from existing code in the same file. I'll update all the old style attributes in the file.

Comment (roseundy) [reply assets/app/view/game/hex.rb:69]: So, strokeWidth: FRAME_COLOR_STROKE_WIDTH doesn't actually work.

Comment (roseundy) [reply assets/app/view/game/hex.rb:69]: Not sure SVG styles work the same.

**Pull request merged**